mnefun/_mnefun.py

```diff
@@ -1002,8 +1004,12 @@ def run_sss_locally(p, subjects, run_indices):
     mne.preprocessing.maxwell_filter
     """
     data_dir = op.join(op.dirname(__file__), 'data')
-    cal_file = op.join(data_dir, 'sss_cal.dat')
-    ct_file = op.join(data_dir, 'ct_sparse.fif')
+    if p.cal_file and p.ct_file is None:
```

Comment:
Yes I meant `if p.cal_file is None and p.ct_file is None` somehow what I have is working :confused:  I thought to combine the conditional since both files need to be provided. 

You mean like....
```
    if p.cal_file is None:
        cal_file = op.join(data_dir, 'sss_cal.dat')
    else:
        cal_file = p.cal_file
    if p.ct_file is None:
        ct_file = op.join(data_dir, 'ct_sparse.fif')
    else:
        ct_file = p.ct_file
```
seems a little bulky, no?